docker/bitcoin-regtest/entrypoint.sh

```diff
@@ -19,11 +19,11 @@ cd /opt/coins/blockbook/bitcoin_regtest
 
 echo "Starting blockbook service"
 bin/blockbook \
-    -blockchaincfg=config/blockchaincfg.json \
+    -blockchaincfg=/opt/coins/blockbook/bitcoin_regtest/config/blockchaincfg.json \
     -datadir=/opt/coins/data/bitcoin_regtest/blockbook/db \
     -sync \
     -internal=:19021 \
     -public=:19121 \
-    -certfile=cert/blockbook \
```

Comment:
Aaah so that's all that was needed to disable HTTPS? Aaah 🤦

cc @karliatto maybe you know already but after we merge this, trezor-user-env will contain this change and blockbook will be accessible via plain HTTP.